kube-router.go

```diff
@@ -10,10 +10,20 @@ import (
 
 	"github.com/cloudnativelabs/kube-router/app"
 	"github.com/cloudnativelabs/kube-router/app/options"
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/pflag"
 )
 
 func main() {
+	if err := Main(); err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", err)
+		os.Exit(1)
+	}
+	os.Exit(0)
```

Comment:
This change (splitting main into 2 functions) was so that I could call `main` in the test and not have it `os.Exit()` the test.